ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessService.scala

```diff
@@ -284,6 +294,11 @@ class DBProcessService(managerActor: ActorRef,
     }
   }
 
+  private def validateInitialScenario(canonicalProcess: CanonicalProcess, processingType:ProcessingType, category: String) = {
+    val validationResult = processValidation.processingTypeValidationWithTypingInfo(canonicalProcess, processingType, category)
+    validationResult.errors.globalErrors.filter(_.typ != ReflectUtils.simpleNameWithoutSuffix(EmptyProcess.getClass))
```

Comment:
wtf? maybe we should replace it with inclusive logic instead of exclusive - I mean filter only specific error classes, not filter out this one class. why this one? Will it be always true? When we will add another "or _.type !=..." in this expression?

Reply:
I'm thinking about sth like:
- displaying only properties errors here
- recognize properties errors with `$properties` (not nice, but we do it already), have to change `ProcessValidation.formatErrors`